pkg/kubelet/dockershim/helpers_linux.go

```diff
@@ -136,8 +136,8 @@ func (ds *dockerService) updateCreateConfig(
 	return nil
 }
 
-func (ds *dockerService) determinePodIPBySandboxID(uid string) string {
-	return ""
+func (ds *dockerService) determinePodIPBySandboxID(uid string) []string {
+	return []string{}
```

Comment:
Just curiosity, why are you returning `nil` in pkg/kubelet/dockershim/docker_sandbox.go , and `[]string{}` here and in other parts?

Reply:
changed, all return `nil` now.